airflow/www/static/css/flash.css

```
@@ -48,3 +49,25 @@
 .expanded-error {
   height: 100%;
 }
+
+.dag-import-error::after {
+  /* symbol for "opening" panels */
+  font-family: FontAwesome;/* stylelint-disable-line font-family-no-missing-generic-family-keyword */
+  content: "\f078";
+  float: right;
+  color: #e43921;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.dag-import-error.expanded-error::after {
+  /* symbol for "closing" panels */
+  font-family: FontAwesome;/* stylelint-disable-line font-family-no-missing-generic-family-keyword */
+  content: "\f054";
+  float: right;
+  color: #e43921;
+  position: absolute;
+  top: 0;
+  right: 0;
```

Seems like most of this would already be inherited from `.dag-import-error`?
```suggestion
  content: "\f054";
```